src/app/loadout/loadout-ui/PlugDef.tsx

```diff
@@ -20,16 +19,14 @@ export default function PlugDef({ plug, onClick, onClose }: Props) {
   const showTooltip = defs && !isPhonePortrait;
 
   const contents = (
-    <div className={styles.emptyItem}>
```

Comment:
I made this change in another PR too, I don't think this empty item class is needed from my testing.